contracts/utils/Signer.sol

```diff
@@ -23,7 +23,7 @@ import "@gnosis.pm/safe-contracts/contracts/interfaces/ISignatureValidator.sol";
  */
 contract Signer is ISignatureValidator {
     // SeedFactory.deploySeed() byte hash
-    bytes4 internal constant SEED_FACTORY_MAGIC_VALUE = 0xda235e6e;
+    bytes4 internal constant SEED_FACTORY_MAGIC_VALUE = 0x0af5d6bb;
```

Comment:
Would changing the Signer and Seed contract have implications for the audit that has been done on them? Just wondering